### ISSUE TITLE
toODataString serialization issue

### ISSUE THREAD

**Issue by chaseguggy** [telerik/kendo-angular#2863]
## Current behavior
Currently, toODataString() does not fix filter operators like process() does for the grid data.
Our team is using OData to do server side filtering on kendo grids. 
When given the filter operator 'equals', the current implementation of the code will error out when trying to create a serialized query string for OData operations. That is because it is actually trying to look for the operator 'eq' to create the string.

This is because the toODataString does use the operatorMap that is in filter.operator.js.
process(), used for client-side paging however does. And will fix the value before applying the operation.

## Expected behavior
What I would expect toODataString to do. Is make use of the same correcting that process() does.
So my filters in the State object can use either 'equals' or 'eq'.
This stumped me for a bit, and would be an easy fix. I think changing this would save people some time. And make the functions used for server-side and client-side paging, uniform. 

## Minimal reproduction of the problem with instructions
[https://stackblitz.com/edit/angular-a8rdk3](https://stackblitz.com/edit/angular-a8rdk3)
In this example, you can see changing 'equals' to 'eq' will make it stop erroring.

TLDR -> toODataString() doesn't understand 'equals', like process() does.

**Comment by dtopalov** [comment]
The supported filter descriptor operators are as follows:

https://www.telerik.com/kendo-angular-ui/components/dataquery/api/FilterDescriptor/#toc-operator

As this tracker is for bug reports only, if you would like us to consider enhancing the current functionality to support other operators (like "equals") in the **toODataString()** function, please submit a feature request to our [Feedback portal ](https://feedback.telerik.com/kendo-angular-ui).

Thank you in advance.

**Comment by tsvetomir** [comment]
We decided to resolve the operator aliases in `toODataString` as suggested for consistency. The update has been released in the latest [development build](https://www.telerik.com/kendo-angular-ui/components/installation/development-builds/).

Thank you for reporting this issue.
